nemo/collections/tts/modules/audio_codec_modules.py

```diff
@@ -2241,6 +2325,49 @@ def forward(self, audio, audio_len):
         spec, spec_len = self.preprocessor(input_signal=audio, length=audio_len)
         return spec, spec_len
 
+class STFTProcessor(NeuralModule):
```

Comment:
Can you docstring this?
Do we not have an equivalent version of this class somewhere?

Reply:
Logic like this is duplicated in many TTS files. I could move this class into a util and refactor newer files (eg. audio_codec_loss.py) to share the definition.

The only similar class outside of the TTS collection is in the audio collection https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/audio/modules/transforms.py#L25